actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -212,10 +211,10 @@ def no_routes(routes)
             <<~MESSAGE
             You don't have any routes defined!
 
-            Please add some routes in config/routes.rb.\n
+            Please add some routes in config/routes.rb.
             MESSAGE
           else
-            "No routes were found for this controller\n"
+            "No routes were found."
           end
           @buffer << "For more information about routes, see the Rails guide: http://guides.rubyonrails.org/routing.html."
         end
```

Comment:
Seems like we should figure out a way to share the 2 `no_routes` methods.

Reply:
Yeah. I'll create `Base` class and inherit from it.